packages/Webkul/Core/src/Providers/CoreServiceProvider.php

```diff
@@ -39,6 +39,7 @@ public function boot(): void
         $this->publishes([
             dirname(__DIR__) . '/Config/concord.php' => config_path('concord.php'),
             dirname(__DIR__) . '/Config/scout.php'   => config_path('scout.php'),
+            dirname(__DIR__) . '/Config/bagisto_filesystem.php'   => config_path('filesystem.php'),
```

Comment:
No need for the extra configuration we can handle this by simple default laravel configuration.

In this way `Storage::url()` is much cleaner rather than chaining with the disk.